bugsnag-android-unity/src/main/java/com/bugsnag/android/unity/UnityClient.java

```diff
@@ -0,0 +1,56 @@
+package com.bugsnag.android.unity;
+
+import java.lang.String;
+
+import android.content.Context;
+
+import com.bugsnag.android.Bugsnag;
+import com.bugsnag.android.Callback;
+import com.bugsnag.android.Client;
+import com.bugsnag.android.Error;
+import com.bugsnag.android.MetaData;
+import com.bugsnag.android.Report;
+import com.bugsnag.android.Severity;
+
+public class UnityClient {
+
+    public static void init(Context androidContext, String apiKey) {
+        Bugsnag.init(androidContext, apiKey);
+    }
+
+    public static void notify(String name, String message, String context, StackTraceElement[] stacktrace, Severity severity, String logLevel) {
+        Bugsnag.notify(name, message, stacktrace, new UnityCallback(context, severity, logLevel));
+    }
+}
+
+class UnityCallback implements Callback {
+    static final String NOTIFIER_NAME = "Bugsnag Unity (Android)";
+    static final String NOTIFIER_VERSION = "3.4.0";
+    static final String NOTIFIER_URL = "https://github.com/bugsnag/bugsnag-unity";
+
+    final private Severity severity;
+    final private String context;
+    final private String logLevel;
```

Comment:
Great that it's final. The Java convention is generally to have the access modifier first i.e. `private final`
https://docs.oracle.com/javase/specs/jls/se8/html/jls-8.html#jls-8.3.1

Reply:
Ack. Good catch!